### PR TITLE
raw messages have an UUID

### PR DESCRIPTION
**Pull request by sapiens-sapide**
raw message id are generated by the application, no more relying on the external message id.

### CODE REVIEW

File: src/backend/main/py.main/caliopen_main/message/core/raw.py [hunk 1]
```diff
@@ -22,19 +23,19 @@ class RawMessage(BaseUserCore):
...
-    _pkey_name = 'raw_id'
+    _pkey_name = 'raw_msg_id'
 
     @classmethod
-    def create(cls, user, message_id, raw):
+    def create(cls, user, raw):
         """Create raw message."""
-        key = hashlib.sha256(message_id).hexdigest()
-        return super(RawMessage, cls).create(user, raw_id=key, data=raw)
+        key = uuid.uuid4()
```
Comment (gdchamal) [inline src/backend/main/py.main/caliopen_main/message/core/raw.py]: we can avoid generating the raw_msg_id uuid as the store layer have a default value generating it.

In a general manner storage layer is responsible of generating ids, i'm ensure having done that everywhere correctly (Discussion for example)

Comment (sapiens-sapide) [inline src/backend/main/py.main/caliopen_main/message/core/raw.py]: uuid.uuid4() still in use in : 

* core/contact
* core/thread
* core/message

you cleanup or I cleanup ?

Comment (sapiens-sapide) [inline src/backend/main/py.main/caliopen_main/message/core/raw.py]: OK, so you're OK to merge this PR ?

Comment (gdchamal) [inline src/backend/main/py.main/caliopen_main/message/core/raw.py]: yes let's merge ;)

**Pull request merged**